setup.py

```diff
@@ -350,9 +350,9 @@ def build(self):
         # Fetch all c and cpp files from the subdirs to compile.
         for library_name in self._sub_library_names:
             for extension in ["*.c", "*.cpp"]:
-                self._source_files.extend(glob.glob(
+                self._source_files.extend(sorted(glob.glob(
```

Comment:
why sort here and not on line 357? what if the order of self._sub_library_names is different between builds?